src/App.jsx

```diff
@@ -14,70 +16,95 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const checkSelectingGood = (row) => {
+    if (selectedGood) {
+      const selected = document.querySelector(
+        '.has-background-success-light',
+      );
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      if (selected !== row) {
+        selected.classList.remove(
+          'has-background-success-light',
+        );
+      }
+    }
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  const selectItem = (e) => {
+    const row = e.target.closest('tr');
```

Comment:
Avoid using closest method directly on DOM elements. Instead, you can pass the needed data directly to the function.